account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/Stamper.java

```diff
@@ -81,24 +58,14 @@ private Stamper() {
         this.bundlePrivateKey = null;
     }
 
-    /**
-     * Creates a new TEK via HpkeTEKManager or returns the existing one
-     *
-     * @return TEK public key
-     */
-    public String initTek() throws GeneralSecurityException, InvalidProtocolBufferException {
-        byte[] tekPublicKeyBytes = this.tekManager.createTEK().getPublicKeyBytes().toByteArray();
-        return Hex.encode(tekPublicKeyBytes);
-    }
-
     /**
      * Injects a credential bundle (Base58-encoded) after decrypting.
      *
      * @param bundle
      *            Base58-encoded credential bundle
      *
      * @throws GeneralSecurityException
-     *             iif initTek is never called.
+     *             if initTek is never called.
      */
     public void injectCredentialBundle(String bundle) throws GeneralSecurityException, InvalidProtocolBufferException {
```

Comment:
Is there a use-case for using this class without the bundle? Can we make it part of the constructor too and move the methods to get public key to TekManager itself?